### DISCUSSION TITLE
ScrollArea: is it possible to change the wheel delta?

### DISCUSSION THREAD

**Discussion by Toshinaki** [radix-ui/primitives#1409] [Help]
I have a list of items with fixed height.

When I set the item height to 50px, the scroll feels smooth, cause one scroll down moves the content up 100px I guess.

Now the item height is 56px, but one scroll stays the same.

How could I change the wheel delta to make one scroll to be the multiple of the item height?

**Comment by benoitgrelard** [comment]
Our `ScrollArea` does not change how scrolling works in any way, it only hooks onto native scroll events, hides the native scrollbar and shows its own UI instead.

If you wanted to change how fast scrolling happens, I suppose anything you can do natively should also work here.
